### PR TITLE
VS-1490 - Refactor python code from extract dir into a scripts directory.

### PR DESCRIPTION
**Pull request by gbggrant**
Refactor python code from extract dir into a scripts directory.

Passing Integration Test [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/f85602d0-6dc5-49d6-82d1-eb58e9966021)
Passing VAT Creation work [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/ddc7fcf9-5fb7-44e2-8117-721389d4f858), [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/0d705f21-3362-4890-b925-5bed2646fe4d), and [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Quickstart%20v3%20ggrant/job_history/15cfe125-e700-44c8-b9d0-c3e98d7db4c0)

### CODE REVIEW

File: scripts/variantstore/scripts/test/test_build_docker_tag.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import re
 import unittest
-from scripts.variantstore.scripts.build_docker_tag import build_tag, build_argument_parser
+from build_docker_tag import build_tag, build_argument_parser
```
Comment (gbggrant) [inline scripts/variantstore/scripts/test/test_build_docker_tag.py:3]: Thanks. I guess that makes sense!

File: scripts/variantstore/scripts/hail_create_vat_inputs.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from typing import Union
 from datetime import datetime
 
-import create_vat_inputs
+from scripts.variantstore.scripts import create_vat_inputs
```
Comment (mcovarr) [inline scripts/variantstore/scripts/hail_create_vat_inputs.py:6]: is this correct? I wouldn't have expected that `scripts.variantstore` bit

Comment (gbggrant) [reply scripts/variantstore/scripts/hail_create_vat_inputs.py:6]: Hmm. Good find. I haven't tested creating the vat - guessing that would fail?

Comment (gbggrant) [reply scripts/variantstore/scripts/hail_create_vat_inputs.py]: Okay, that *was* incorrect. Working now.

**Review by RoriCremer — APPROVED** [review]

**Pull request merged**